yt_dlp/extractor/facebook.py

```diff
@@ -616,6 +603,32 @@ def parse_graphql_video(video):
                                 'url': playable_url,
                             })
                     extract_dash_manifest(video, formats)
+
+                    automatic_captions, subtitles = {}, {}
+                    is_video_broadcast = get_first([video], 'is_video_broadcast', expected_type=bool)
```

Comment:
`get_first(x, y)` is just `traverse_obj(x, (..., y), get_all=False)`
```suggestion
                    is_video_broadcast = traverse_obj(video, ('is_video_broadcast', {bool}))
```